### ISSUE TITLE
intermittent error running terraform apply error on docs/examples/networking/nat/nat.tf

### ISSUE THREAD

**Issue by jlamillan** [oracle/terraform-provider-oci#319]
**UPDATE** this issue appears intermittent.

### Terraform Version

Terraform v0.10.7

### OCI Provider Version

2017/10/19 11:03:24 [INFO] terraform-provider-oci v2.0.0 (same behavior with v2.0.2 as well)

### Description:

The NAT example, docs/examples/networking/nat/nat.tf appears to be broken out of the box. I am trying to run it without modification and leaving all the network variables (*cidr) to their defaults and I am using OEL 7.4 as the OS.

### Terraform Apply

Apply creates a number of resources, including the vcn and the NatInstance instance, but eventually fails when creating the `oci_core_route_table.PrivateRouteTable` with the following error:

```
....
Error applying plan:

1 error(s) occurred:

* oci_core_route_table.PrivateRouteTable: 1 error(s) occurred:

* oci_core_route_table.PrivateRouteTable: Status: 400; Code: InvalidParameter; OPC Request ID: /OMITTED; Message: PrivateIP {ocid1.privateip.oc1.iad.OMITTED} is an invalid route target. {The Private IP is attached to a VNIC whose SRC/DST check is enabled.}.

Terraform does not automatically rollback in the face of errors.
Instead, your Terraform state file has been partially updated with
any resources that successfully completed. Please address the error
above and apply again to incrementally change your infrastructure.
```

**Comment by briangustafson** [comment]
Hi - I just tried downloading the 2.0.2 and applying the nat example, and it worked for me. I set only the variables in env-vars, except for the variables under VCN configuration and Instance Configuration which I deleted from that file. 2.0.0 did not have support for skip_source_dest_check, so the error you saw on that version makes sense. (That version also had a bug where you could add arbitrary keys in create_vnic_details, so you would not get an error when trying to specify that property. That has been fixed in 2.0.0) When you tried in 2.0.2, did you destroy and then apply from the beginning?

**Comment by jlamillan** [comment]
> When you tried in 2.0.2, did you destroy and then apply from the beginning?

No, I didn't. After tearing down completely, I was able to get it with 2.0.2. It'd probably be good to have a comment somewhere that the NAT example requires >= 2.0.2.

Thanks for the feedback.
